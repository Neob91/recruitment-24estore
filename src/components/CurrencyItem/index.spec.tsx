import React from 'react';
import Renderer from 'react-test-renderer';
import { PCurrencyItem } from '.';

describe('CurrencyItem', () => {
  it('renders correctly', () => {
    const tree = Renderer
      .create(<PCurrencyItem code="EUR" name="Euro" mid={4.2} isFavorite={false} onClick={() => null} />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
