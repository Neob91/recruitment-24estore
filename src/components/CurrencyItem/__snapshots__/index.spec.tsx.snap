// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CurrencyItem renders correctly 1`] = `
<div
  className="css-13u28c6"
  onClick={[Function]}
>
  <div>
    EUR
  </div>
  <div>
    Euro
  </div>
  <div>
    4.2
  </div>
</div>
`;
